### ISSUE TITLE
[Exasol] Generated delete scripts wrong

### ISSUE THREAD

**Issue by redcatbear** [dbeaver/dbeaver#9389]
#### System information: 
- Operating system (distribution) and version: Linux n11 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux, Ubuntu 20.4
- DBeaver version: 7.1.3.202007192031 (as Eclipse plug-in, details below)
- Additional extensions: see below

```
  DBeaver IDE	7.1.3.202007192031	org.jkiss.dbeaver.ide.feature.feature.group	JKISS
    DBeaver Databases	1.0.28.202007192031	org.jkiss.dbeaver.db.feature.feature.group	JKISS
    DBeaver Databases UI	1.0.28.202007192031	org.jkiss.dbeaver.db.ui.feature.feature.group	JKISS
    DBeaver Debug	6.1.27.202007192031	org.jkiss.dbeaver.debug.feature.feature.group	JKISS
    DBeaver Desktop UI	7.1.3.202007192031	org.jkiss.dbeaver.ui.feature.feature.group	JKISS
    DBeaver Runtime	7.1.3.202007192031	org.jkiss.dbeaver.runtime.feature.feature.group	JKISS
  DBeaver SVG Extension	1.0.73.202007192031	org.jkiss.dbeaver.ext.ui.svg.feature.feature.group	DBeaver Corp
```

#### Connection specification:
- Database name and version: Exasol 6.2.5
- Driver name: `exasol-jdbc-6.2.5.jar`
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No.

#### Describe the problem you're observing:

When trying to delete objects from withing the database navigator, for some objects the `DROP` statements are syntactically incorrect.

Deleting a Virtual Schema creates

    DROP SCHEMA <virtual-schema-name> CASCADE;

instead of

    DROP [FORCE] VIRTUAL SCHEMA <virtual-schema-name> CASCADE;

Where `FORCE` is optional. See also: https://docs.exasol.com/sql/drop_schema.htm?Highlight=drop%20virtual%20schema

Deleting an adapter script creates

    DROP SCRIPT <schema-name>.<script-name>;

instead of

    DROP ADPATER SCRIPT <schema-name>.<script-name>;

See also: https://docs.exasol.com/sql/drop_script.htm

Unfortunately without the missing parts the `DROP` command try to drop a regular schema and a regular script instead.

#### Steps to reproduce, if exist:

* Open view "Datebase Navigator"
* Create connection to an Exasol database
* Create a schema to host the adapter script
* [Create an adapter script](https://github.com/exasol/virtual-schemas/blob/master/doc/user-guide/user_guide.md#deploying-the-adapter-script) inside the schema you created
* [Create a virtual schema](https://github.com/exasol/virtual-schemas/blob/master/doc/user-guide/user_guide.md#deploying-the-adapter-script)
* Find the virtual schema in the navigator
* Right click the virtual schema
* Select "Delete"
* Click "View Script"
* Notice the wrongly generated script
* Delete the virtual schema by hand using the SQL editor
* Find the adapter script in the navigator
* Right click the adapter
* Select "Delete"
* Click "View Script"
* Notice the wrongly generated script

#### Include any warning/errors/backtraces from the logs

`schema <virtual-schema-name> is a virtual schema. Please use DROP VIRTUAL SCHEMA instead. [line 1, column 13] (Session: 1673528044707184640)`

`script <adapter-script-name> is an adapter script. Please use DROP ADAPTER SCRIPT instead. [line 1, column 13] (Session: 1673528044707184640)`

**Comment by kseniiaguzeeva** [comment]
Thank you for the bug report.

**Comment by Sargul** [comment]
I'll have a look at this.

Virtual Schema support is not quite complete

**Comment by Sargul** [comment]
Fixed in PR #9434

![image](https://user-images.githubusercontent.com/5595511/89191141-ce40e880-d5a2-11ea-91e1-3041b3ad912c.png)
![image](https://user-images.githubusercontent.com/5595511/89191195-def15e80-d5a2-11ea-9b94-024e9f128041.png)
